nix/noninteractive.nix

```diff
@@ -20,7 +20,9 @@
   nix.registry = lib.mkForce { };
 
   # would pull in nano
-  programs.nano.syntaxHighlight = lib.mkForce false;
+  programs.nano.enable = false;
```

Comment:
I also cherry picked this one into main already.

Reply:
https://github.com/nix-community/nixos-images/pull/211/commits/c54ae6f4d1ed6303f33b61f39a0fef9d6c58bff8